pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -857,22 +857,24 @@ public static void deactivateKitRequest(int dsmKitRequestId, @NonNull String dea
                     results.resultException);
         }
         if (Objects.nonNull(ddpInstanceDto)) {
-            KitRequestShipping kitRequestShipping = new KitRequestShipping();
-            kitRequestShipping.setDsmKitRequestId(dsmKitRequestId);
-            kitRequestShipping.setDeactivationReason(deactivationReason);
-            kitRequestShipping.setDeactivatedDate(deactivatedDate);
-            kitRequestShipping.setKitTypeName(kitRequestDao.getKitTypeByKitRequestId(dsmKitRequestId));
-            try {
-                //todo: if multiple kits exist for same kit-request(dsmKitRequestId), we are updating all elements? edge case though
-                UpsertPainlessFacade.of(DBConstants.DDP_KIT_REQUEST_ALIAS, kitRequestShipping, ddpInstanceDto,
-                        ESObjectConstants.DSM_KIT_REQUEST_ID, ESObjectConstants.DSM_KIT_REQUEST_ID, dsmKitRequestId,
-                        new PutToNestedScriptBuilder()).export();
-            } catch (Exception e) {
-                logger.error(String.format("Error updating kit request shipping deactivate reason with dsm kit request id: %s in "
-                        + "ElasticSearch", dsmKitRequestId));
-                e.printStackTrace();
+            if (ddpInstanceDto.isESUpdatePossible()) {
+                //skip if no elastic index. ex: juniper studies/eating-disorder
+                KitRequestShipping kitRequestShipping = new KitRequestShipping();
+                kitRequestShipping.setDsmKitRequestId(dsmKitRequestId);
+                kitRequestShipping.setDeactivationReason(deactivationReason);
+                kitRequestShipping.setDeactivatedDate(deactivatedDate);
+                kitRequestShipping.setKitTypeName(kitRequestDao.getKitTypeByKitRequestId(dsmKitRequestId));
+                try {
+                    //todo: if multiple kits exist for same kit-request(dsmKitRequestId), we are updating all elements? edge case though
+                    UpsertPainlessFacade.of(DBConstants.DDP_KIT_REQUEST_ALIAS, kitRequestShipping, ddpInstanceDto,
+                            ESObjectConstants.DSM_KIT_REQUEST_ID, ESObjectConstants.DSM_KIT_REQUEST_ID, dsmKitRequestId,
+                            new PutToNestedScriptBuilder()).export();
+                } catch (Exception e) {
+                    logger.error(String.format("Error updating kit request shipping deactivate reason with dsm kit request id: %s in "
+                            + "ElasticSearch", dsmKitRequestId));
+                    e.printStackTrace();
```

Comment:
The exception message should be included in the logged error.

Reply:
Its existing code and we have a e.printStackTrace() right below but I agree its helps to have exception message in alert. 
Added to the error log so that we get the message as part of the alert

Reply:
BTW, you do not need `String.format` in log messages since '{}' works and makes the code easier to read.